packages/block-editor/src/components/block-toolbar-link-control/compute-nice-url.js

```diff
@@ -0,0 +1,23 @@
+export default function computeNiceURL( url ) {
+	if ( ! url ) {
+		return '';
+	}
+
+	let urlData;
```

Comment:
Const needs to be initialized so it could be one only if I moved the rest of the code to `try { } catch { }` which I'm pretty hesitant of as it would hide any unintended exceptions.